.github/SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security policy
+
+1. [Supported Versions](#1-supported-versions)  
+2. [Vulnerability Report](#2-vulnerability-report)  
+3. [Security Disclosure](#3-security-disclosure)  
+
+---
+
+## 1. Supported Versions
+
+We are releasing patches to eliminate vulnerabilities, you can see below:
+
+| Version     | Supported by |
+| ----------- | ------------ |
+| 1.0.x       | N/A          |
+
+---
+
+## 2. Vulnerability Report
+
+The Edge-Orchestration team assigns the highest priority to all security bugs in Edge-Orchestration project. We appreciate your efforts and
+responsible disclosure of information to eliminate vulnerabilities.
+
+Please report security bugs by emailing the Lead Maintainer at moonki1.hong@samsung.com or Security Advisor at t.drozdovsky@samsung.com marked "SECURITY".
```

Comment:
The dedicate security account for HomeEdge has been created: homeedge-security-issues@lists.lfedge.org

Reply:
@MoonkiHong Update done! Please review!